pkgs/games/fltrator/default.nix

```diff
@@ -0,0 +1,43 @@
+{ stdenv, fetchurl, unzip, fltk, which, libjpeg }:
+
+stdenv.mkDerivation rec {
+  name = "fltrator-${version}";
+  version = "2.3";
+  src = fetchurl {
+    url = "https://sourceforge.net/projects/fltrator/files/v${version}/fltrator-${version}-code.zip/download";
+    sha256 = "125aqq1sfrm0c9cm6gyylwdmc8xrb0rjf563xvw7q28sdbl6ayp7";
+  };
+
+
+  buildInputs = [ fltk unzip which libjpeg ];
+
+  unpackPhase = ''
+    unzip $src
+    '';
+
+  configurePhase = ''
+    cd fltrator*
+    sed -i "s%home += \"fltrator/\"%home = \"$out/fltrator/\"%" src/fltrator.cxx
+    sed -i "s%home += \"fltrator/\"%home = \"$out/fltrator/\"%" src/fltrator-landscape.cxx
```

Comment:
`substituteInPlace` might be slightly more ergonomic here: https://nixos.org/nixpkgs/manual/#ssec-stdenv-functions

Also, this does not make it try and write to `$out` at runtime, right?

Reply:
config are saved in an fltk-dependant path, but the level editor save modified level in the out path. I don't know how to solve this. I suppose I can left this in this state.

Reply:
Alright, perhaps that can come later. :) Normally that path will be read-only, and some patching may be required to cause the game to save it in the user's homedir or something like that.